src/main.cpp

```diff
@@ -4727,12 +4727,10 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
         if (pfrom->nVersion >= NO_BLOOM_VERSION) {
             Misbehaving(pfrom->GetId(), 100);
             return false;
+        } else if (GetBoolArg("-enforcenodebloom", false)) {
```

Comment:
All the other options have symbolic constants for their default; this should too.